lib/lndclient/LndClient.ts

```diff
@@ -53,7 +53,7 @@ class LndClient extends BaseClient {
   private credentials!: ChannelCredentials;
   private reconnectionTimer?: NodeJS.Timer;
   private identityPubKey?: string;
-
+  private invoiceSubscription!: ClientReadableStream<lndrpc.InvoiceSubscription>;
```

Comment:
I don't think we need the `!` operator here, but rather the nullable `?` operator to indicate that sometimes we won't have an active invoice subscription. You already have null checks where this is being used, which is good, so I don't think you'd need to change anything else.